src/Symfony/Component/Yaml/Inline.php

```diff
@@ -398,7 +398,7 @@ private static function parseSequence($sequence, $flags, &$i = 0, $references =
                     $value = self::parseScalar($sequence, $flags, array(',', ']'), array('"', "'"), $i, true, $references);
 
                     // the value can be an array if a reference has been resolved to an array var
-                    if (!is_array($value) && !$isQuoted && false !== strpos($value, ': ')) {
+                    if (!is_array($value) && !$value instanceof \DateTimeInterface && !$isQuoted && false !== strpos($value, ': ')) {
```

Comment:
That might also do the job, of course, but I don't really overlook the consequences. Running `strpos` on an object will most likely fail in most of the situations, so it's probably reasonable, yes.  But as I said, I don't overlook every aspect — are there any kinds of serializable objects allowed? — so I wanted to keep it as specific as possible. Feel free to loosen this in case `is_object()` doesn't break anything I don't see ...